signature/envelope.go

```diff
@@ -0,0 +1,92 @@
+// Package signature provides operations for types that implement
+// signature.Envelope or signature.Signer.
+//
+// An Envelope is a structure that creates and verifies a signature using the
+// specified signing algorithm with required validation. To register a new
+// envelope, call RegisterEnvelopeType first during the initialization.
+//
+// A Signer is a structure used to sign payload generated after signature
+// envelope created. The underlying signing logic is provided by the underlying
+// local crypto library or the external signing plugin.
+package signature
+
+import "fmt"
+
+// Envelope provides functions to basic functions to manipulate signatures.
```

Comment:
I'm not quite sure, what does 'functions to basic functions' mean here. Maybe "Envelope provides basic functions to manipulate signatures." ?